xml/2018-10-01.xml

```diff
@@ -0,0 +1,232 @@
+    bring together the community of students who are interested in Computing.
+
+    **Committee -** This year, DoCSoc is brought to you by:
+
+    - **President**: Inara Ramji
+    - **Vice President**: Alessandro Bonardi
+    - **Treasurer**: Rachel Mekhteiva Lee
+    - **Industrial Events Director**: Rohan Pritchard
+    - **Social Events Director**: Jayati Sarkar
+    - **Academic Events Director**: Jack Pordi
+    - **Secretary**: Kelvin Zhang
+    - **Events Officers**: Szilvester Szekely, Serene Chongtrakul, Anindita (Andy) Ghosh
+    - **Marketing &amp; Publicity Officer**: Fawaz Shah
+    - **Webmaster**: George Soteriou
+    - **GameDev Officer**: Athi Dharma
+
+    **Dep Reps -** In addition to the DoCSoc committee, we have three departmental representatives
+    (dep reps) this year responsible for representing the student body to the department:
+
+    - **Computing**: Martin Zlocha ([email](mailto:dep-rep@doc.ic.ac.uk))
+    - **Joint Maths and Computing (JMC)**: Rachel Mekhtieva Lee and Fawaz Shah ([email](mailto:dep-rep@doc.ic.ac.uk))
+    - **Electronic and Information Engineering (EIE)**: Willem van der Schoot (Academic, [email](willem.van-der-schoot16@imperial.ac.uk)) and Mayank Surana (Wellbeing, [email](mayank.surana16@imperial.ac.uk))
+
+    If you have any academic or wellbeing related queries, your dep reps are people
+    you can talk to.
+
+    **Events -** DoCSoc runs a huge number of events every year. Our main events
+    include:
+
+    - **Talks and Workshops** - Not to be confused with the department's lunchtime
+    ACI talks, DoCSoc runs engaging talks and workshops with our sponsors, usually
+    in the evening with pizza (with dietary requirements catered for) and soft
+    drinks provided. In the past, we've run workshops such as capture the flag competitions
+    with Netcraft, an _Extreme Startup_ event with Palantir and Bloomberg's _CodeCon_.
+
+    - **Bar Nights** - Our most well-known events include our bar nights which
+    usually occur every month during term time. Bar nights are a chance for you to
+    socialise with your coursemates, make new friends and occasionally network with
+    recruiters sponsoring our bar nights. Bar nights are _free entry_ to our members
+    and you will be given tokens for _free_ drinks (both alcoholic and non-alcoholic)
+    to use at the bar!
+
+    - **IC Hack** - Every year, we hold our flagship hackathon, IC Hack. Last year,
+    attendees hacked away for twenty-four hours building their competition entries,
+    while we kept them engaged with workshops from five sponsors, a meditation and
+    sleep class with NextJump, games rooms, karaoke and much more! Check out the
+    montage from our last year [here](https://www.youtube.com/watch?v=Hngr7ts45Dc).
+
+    Do you have suggestions for events? We're always looking for feedback so feel
+    free to email us with anything you have in mind!
```

Comment:
"Throughout the year we also host a wide range of social events (go-karting, cinema and theatre trips) and academic events. Do you have any suggestions for an event you would like to see? We're always...